tests/acceptance/pageObjects/webPage.js

```diff
@@ -81,13 +81,16 @@ module.exports = {
     getNotifications: async function() {
       const notifications = []
       await this.toggleNotificationDrawer()
-      await this.api.elements('@notificationElement', result => {
-        for (const element of result.value) {
-          this.api.elementIdText(element.ELEMENT, text => {
-            notifications.push(text.value)
-          })
+      await this.waitForElementVisible('@notificationElement').api.elements(
```

Comment:
This is the only change that is required for the fix. All the other changes made in this PR (drone and feature file) will be removed after the approval/review.